ext/enchant/enchant.stubs.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/** @return resource|false */
+function enchant_broker_init() {}
+
+/** @resource $broker */
+function enchant_broker_free($broker): bool {}
+
+/** @resource $broker */
+function enchant_broker_set_dict_path($broker, int $dict_type, string $value): bool {}
+
+/**
+ * @param resource $broker
+ * @return string|false
+ */
+function enchant_broker_get_dict_path($broker, int $dict_type) {}
```

Comment:
It might be preferable to stick with the former parameter name `$name` instead of `$dict_type` for BC reasons.